app/javascript/mastodon/features/getting_started/index.js

```diff
@@ -22,6 +22,7 @@ const messages = defineMessages({
   settings_subheading: { id: 'column_subheading.settings', defaultMessage: 'Settings' },
   community_timeline: { id: 'navigation_bar.community_timeline', defaultMessage: 'Local timeline' },
   direct: { id: 'navigation_bar.direct', defaultMessage: 'Direct messages' },
+  bookmarks: { id: 'navigation_bar.bookmarks', defaultMessage: 'Bookmarks' },
```

Comment:
The navigation bar item is missing. You should probably add something like this to `/home/muveexu/donotban/live/app/javascript/mastodon/features/ui/components/navigation_panel.js`
```
<NavLink className='column-link column-link--transparent' to='/bookmarks'><Icon className='column-link__icon' id='bookmark' fixedWidth /><FormattedMessage id='navigation_bar.bookmarks' defaultMessage='Bookmarks' /></NavLink>    
```
And adjust the column height for single-column mode accordingly:
Change [this line](https://github.com/tootsuite/mastodon/blob/c31bc41a02d89eea654f8d7c74c0a1cb544cd919/app/javascript/mastodon/features/getting_started/index.js#L133) to `height += 48*4;`.